duckduckgo/Android
# Hotfix/5.85.3

Task/Issue URL: https://app.asana.com/0/488551667048375/1200295267550935/f
Tech Design URL: 
CC: 

**Description**:
This rollbacks changes introduced in 5.85.2 to mitigate the issue of sites blocked by Akamai. They reverted and fixed the issue on their side.


**Steps to test this PR**:
1. visit any website (you can try some sites that didn't load due to related issue like `accuweather.com`)
1. go to Logcat and filter by `User Agent`
1. Ensure our user agent includes `DuckDuckGo`

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
